### PR TITLE
Refactored com.github.c77 package name to com.ekumen.

### PR DESCRIPTION
**Pull request by jubeira**
Corrected package name in manifest too.

### CODE REVIEW

**Comment by adamantivm** [comment]
why not make it `com.ekumen`? Also, what's the motivation of this change, what does it unblock?

**Comment by jubeira** [comment]
I used ekumenlabs instead of ekumen because of the name of the github organization, perhaps I'm wrong with this one.
It's just a matter of preference actually, it can be undone if it does not apply.

**Comment by jubeira** [comment]
PR closed, no need of this refactoring.

**Comment by jubeira** [comment]
@adamantivm ptal

File: base_controller_lib/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-      package="com.c77.move_base"
+      package="com.ekumen.base_controller"
```
Comment (adamantivm) [inline base_controller_lib/src/main/AndroidManifest.xml:3]: +1

File: build.gradle [hunk 2]
```diff
@@ -27,7 +27,7 @@ apply plugin: 'catkin'
 
 allprojects {
     /* A github url provides a good standard unique name for your project */
```
Comment (adamantivm) [inline build.gradle]: nit: update or remove the comment above, it is now obsolete

**Review by adamantivm — APPROVED** [review]
one minor nit otherwise LGTM

**Comment by jubeira** [comment]
Thanks @adamantivm , corrected & merged :+1:

**Pull request merged**